src/content/eol/2024/04/eol-04-24-24-createtracer.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
We should add a callout to the `createTracer()` doc in the browser section

Reply:
Good call